packages/api-explorer/src/scenes/TypeTagScene/TypeTagScene.tsx

```diff
@@ -43,21 +46,40 @@ interface TypeTagSceneParams {
 
 export const TypeTagScene: FC<TypeTagSceneProps> = ({ api }) => {
   const { specKey, typeTag } = useParams<TypeTagSceneParams>()
-  const navigate = useNavigation()
-  const [value, setValue] = useState('ALL')
+  const { navigate, buildPathWithGlobal, navigateWithGlobal } = useNavigation()
+  const selectedTagFilter = useSelector(selectTagFilter)
+  const { setTagFilterAction } = useSettingActions()
+  const [tagFilter, setTagFilter] = useState(selectedTagFilter)
+  useTagStoreSync()
+
+  const handleChange = (filter: string) => {
+    navigate(location.pathname, {
+      v: filter === 'ALL' ? null : filter.toLowerCase(),
+    })
+  }
 
   useEffect(() => {
-    /** Reset ButtonToggle value on route change */
-    setValue('ALL')
-  }, [typeTag])
+    const searchParams = new URLSearchParams(location.search)
+    let verbParam = searchParams.get('v') || 'ALL'
```

Comment:
`v` doesn't seem very applicable to type tag scenes. Perhaps we should use `f` for filter on this scene instead. @jkaster thoughts?

Reply:
What would we be filtering? `f` would be fine, but I don't know what it's used for. Maybe `t` for type?

Reply:
or `t` for tag, depending on what's being filtered 🤷🏼

Reply:
@patnir41 let's go with `t` for type on type tag scenes.

Reply:
@bryans99 We discussed the above off-thread and came across 2 potential decisions:

1. instead of using `v`, have the parameter be `t` for both MethodTag and TypeTag scenes. 
2. Customize the `tagStoreSync` hook to take a character argument, either `v` or `t`, which will determine which parameter the hook is syncing when rendering a scene.  We then have TypeTag scenes involve logic dealing with a `t` parameter value, and MethodTag scenes involve logic dealing with `v` in the URL.

Do you have any thoughts?

Reply:
Moving forward with renaming parameter from 'v' to 'f' for both scenes.